src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(pyxtp LANGUAGES CXX)
+
+find_package(pybind11 REQUIRED HINTS)
+find_package(VOTCA_XTP REQUIRED NO_MODULE)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds
+ PUBLIC
+ ${PROJECT_SOURCE_DIR}/binds/include
+)
+
+target_compile_options(xtp_binds
+ PUBLIC
+ "-Wall"
```

Comment:
Why? We have an cmake option for this already. Plus not all compilers support `-Wall`.